about/src/main/java/dev/arkbuilders/components/about/Color.kt

```diff
@@ -0,0 +1,15 @@
+package dev.arkbuilders.components.about
+
+import androidx.compose.ui.graphics.Color
+
+internal object ArkColor {
+    val TextPrimary = Color(0xFF101828)
```

Comment:
Do we have these colors in our colors.xml file? If so, can we reuse these values? If not then it's fine

Reply:
These colors are presented only in Compose and are not used in other modules.